src/services/auth/auth-router.ts

```diff
@@ -14,6 +17,37 @@ for (const key in DeviceRedirects) {
 
 const authRouter = Router();
 
+// Add role to userId by email address (admin only endpoint)
+authRouter.put(
+    "/addRoleByEmail/",
+    RoleChecker([Role.Enum.ADMIN]),
+    async (req, res, next) => {
+        try {
+            const email: string = req.body.email as string;
+            const role: string = req.body.role as string;
```

Comment:
create a zod schema for this (can call it `AuthRoleChangeRequest` or something, and then pass request body through it)